CHANGELOG.md

```diff
@@ -1,3 +1,11 @@
+## Unreleased
+
+* [Enhancement] If `S3_PROFILE_IMAGE_BUCKET` is not set, store profile 
+  images to the filesystem. This is the new default behavior. If a bucket 
```

Comment:
*in* the filesystem.

And also, "this is the new default behavior" is a bit misleading; I'd change this to "this matches Tutor's default behavior (when running without plugins)" or something similar.

Reply:
Done